### PR TITLE
fixes #9110 - add freeip route to API

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: app/controllers/api/v2/subnets_controller.rb [hunk 1]
```diff
@@ -70,8 +71,31 @@ def destroy
         process_response @subnet.destroy
       end
 
+      api :GET, "/subnets/:id/freeip", N_("Provides an unused ip address in this subnet")
+      param :id, :identifier, :required => true
+      param :mac, String, :desc => N_("MAC address to reuse the ip for this host")
```
Comment (domcleal) [inline app/controllers/api/v2/subnets_controller.rb]: Upper case "IP" here

File: app/controllers/api/v2/subnets_controller.rb [hunk 2]
```diff
@@ -70,8 +71,31 @@ def destroy
...
+      param :excluded_ips, Array, :desc => N_("IP Addresses that should be excluded from suggestion")
```
Comment (domcleal) [inline app/controllers/api/v2/subnets_controller.rb]: and lower case "addresses" here

**Comment by timogoebel** [comment]
@domcleal : Fixed the spelling. Thanks!

**Comment by domcleal** [comment]
Merged as 83ee735, thanks @timogoebel.

**Pull request closed without merging**